app/models/film.rb

```diff
@@ -5,6 +5,9 @@ class Film < ApplicationRecord
   has_one_attached :film_link, dependent: :purge_later
   validates :title, presence: true
   validate :check_cover
+  extend FriendlyId
```

Comment:
Please move this line at the top of Film class
IMO it's more readable when all extends, includes, requires are there